Tests/AT001.New-ResourceGroup.Acceptance.Tests.ps1

```diff
@@ -24,6 +24,12 @@ Describe "New-ResourceGroup Tests" -Tag "Acceptance-ARM" {
         $Result = Get-AzureRMResourceGroup -Name $ResourceGroupName -Location "$Config.location"
         $Result.location | Should Be $Config.location.Replace(" ","").ToLower()
     }
+
+    It "Should contain a Resource Group with Tags" {
+        $Result = Get-AzureRmResourceGroup -Name $ResourceGroupName -ErrorAction SilentlyContinue
+        $Result.Tags.Count | Should Be -ge 1
```

Comment:
This syntax is incorrect, need to use 'Should Be' or 'Should BeGreaterThan' https://github.com/pester/Pester/wiki/Should

We also need more tests to cover the different code paths